### ISSUE TITLE
still active?

### ISSUE THREAD

**Issue by toddself** [devdazed/boxcar-api#8]
is this project still active? the last update is ~4 years ago

**Comment by devdazed** [comment]
no, :( and I'm not entirely sure boxcar is still a thing.  if it has, it most likely will have severely deviated from the API this was written against.

**Comment by toddself** [comment]
Bj

**Comment by toddself** [comment]
I am working on a new project that I would love to publish under boxcar on npm. https://github.com/toddself/boxcar

Would you be willing to change the npm name for this? I can redirect anyone who was trying to find this happily to the new name

**Comment by devdazed** [comment]
sure, ill just redirect from here to yours also, what is your username on npmjs?

**Comment by toddself** [comment]
Toddself as well. Thanks much!

**Comment by devdazed** [comment]
ok, added, let me know if you have any issues.

**Comment by toddself** [comment]
Awesome.  Thank you so much!
